code/scripting/api/libs/ui.cpp

```diff
@@ -469,8 +465,7 @@ ADE_FUNC(acceptPilot, l_UserInterface_Barracks, "player selection", "Accept the
 ADE_LIB_DERIV(l_UserInterface_Options,
 	"OptionsMenu",
 	nullptr,
-	"API for accessing values specific to the options screen.<br><b>Warning:</b> This is an internal "
-	"API for the new UI system. This should not be used by other code and may be removed in the future!",
+	"API for accessing values specific to the options screen.",
```

Comment:
Are `Screen` and `UI` the same in these comments? Some say `options screens` while others say `options UI`. A few screen/UI names are also capitalized? Sorry if this is very minor, just wanted to double check now rather than later.

Reply:
Eh, might as well be consistent. Done.